testdata/Expecto.Focused.Tests/.gitignore

```diff
@@ -0,0 +1,2 @@
+bin
```

Comment:
Didn't wanted to change the root .gitignore because in testdata can enter in the future something exotic - tests related